chart/values.yaml

```
@@ -8,21 +18,14 @@ cartoConfigValues:
   cartoAuth0ClientId: ""
   cartoAuth0CustomDomain: ""
   cartoSelfhostedDwLocation: ""
-  httpCacheEnabled: "true"
-  storageProvider: "gcp"
-  importBucket: ""
-  routerSslAutogenerate: ""
-  selfHostedDomain: "carto3-selfhosted.lan"
   selfHostedGcpProjectId: ""
   selfHostedTenantId: ""
-  workspaceImportsBucket: ""
-  workspaceThumbnailsBucket: ""
-  workspaceThumbnailsPublic: ""
   reactAppLaunchDarklyClientSideId: ""
-  gcsBucketsProjectId: ""
 
-cartoSecrets:
+clientSecrets:
   googleMapsApiKey: ""
+
+cartoSecrets:
```

I think that the name should be:
```suggestion
cartoSecretsValues:
```